com.one_ware.OneWare.metainfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.one_ware.OneWare</id>
+  <name>OneWare Studio</name>
+  <summary>IDE for Electronics Development</summary>
+  <developer id="one-ware.com">
+    <name translatable="no">Hendrik Mennen</name>
+  </developer>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
you said " It is a free software " but then this. Which is it?

Reply:
I think I said this wrong, I appologize. By free I wanted to say free to use, unlike many Jetbrains IDEs for example where you need to login first. The source is not entirely free (yet), which we might do at some point. A big part of the IDE system is already open source, but not everything.